CoroutineTemplate/build.gradle

```diff
@@ -0,0 +1,77 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext {
+        build_gradle_version = '4.1.1'
+
+        android_version_code = 9
+        android_version_name = "0.7.0"
+        android_min_sdk_version = 21
+        android_target_sdk_version = 28
+        android_compile_sdk_version = 28
+
+        android_support_version = '1.1.0'
+        constraint_layout_version = '2.0.0-alpha3'
+        androidx_core_ktx_version = '1.3.0'
+
+        kotlin_version = '1.4.30'
+        kotlinx_coroutines_version = '1.4.1'
+
+        android_test_junit_version = '1.1.1'
+        test_espresso_version = "3.1.0"
+        test_junit_version = '4.13'
+    }
+
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:$build_gradle_version"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+plugins {
+    id "io.gitlab.arturbosch.detekt" version "1.14.0"
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+        maven { url 'https://jitpack.io' }
+        mavenCentral()
+    }
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
+
+detekt {
+    toolVersion = "1.14.0"
+
+    input = files("app/src/main/java", "app/src/main/kotlin", "data/src/main/kotlin", "data/src/main/java")
```

Comment:
I think we only have `java` source folder

Reply:
@luongvo Should we remove `"app/src/main/kotlin"` in JavaTemplate as well 🤔 cc @Tuubz

Reply:
I agree

- For `CoroutineTemplate` we can keep it as: `"app/src/main/java"`
- For `RxJavaTemplate` we can keep it as: `"app/src/main/java", "data/src/main/java"`

What do you think?

Reply:
Just re-checked, no kotlin source folder is being used, you can remove it 👌 

2 places
![image](https://user-images.githubusercontent.com/16315358/111730759-a8075900-88a4-11eb-9363-2727c7ae311a.png)